modules/ossm-release-3-0-TP-1.adoc

```diff
@@ -0,0 +1,59 @@
+////
+Module included in the following assemblies:
+* service-mesh-docs-main/ossm-release-notes-assembly.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-3-0-TP-1_{context}"]
+= {SMProductName} 3.0 Technology Preview
+
+This release of {SMProductName} is Technology Preview, and is not recommended for use in production environments.
+
+// Simple table to help clarify for users the component versions they can use with OSSM 3.0 TP1. Likely a more complicated table will be needed for GA. OSSM 3.0 separates all the Operators such as OCP, Istio, Envoy, Kiali, OTEL/Tempo, etc. This is only what is known to work with OSSM 3.0 TP1.
+
+[id="ossm-release-3-0-TP1-component-versions_{context}"]
+== Component versions
+
+|===
+|Component |Version
+
+|{ocp-product-title}
+|14+
+
+|Istio
+|1.23.0
+
+|Envoy Proxy
+|1.31
+
+|Kiali Operator
+|1.89
+
+|Kiali Server
+|1.89
+|===
+
+[id="ossm-release-3-0-TP1-features-not-supported_{context}"]
+== Features not currently supported
+
+The following features are not supported in this Technology Preview release:
+
+* {SMPlugin}
+* Istio's Ambient mode
+* Istio's Virtual Machine Architecture
+
```

Comment:
Can we also add the [default tag](https://istio.io/latest/docs/setup/upgrade/canary/#default-tag) to this list? Unlike upstream Istio, the Sail Operator does not, yet, create a default tag with deployments. This means that:

1. Validation only works in namespaces with the `istio.io/rev` label.
2. The `istio-injection: enabled` label only works if you name your `Istio` resource `default`.

I know this is a barebones list but IMO those are big enough behavior changes from upstream Istio that are worth calling out.

Reply:
Do points 1 and 2 need to be included, or does only "Istio's `default` tag" need to be added?

Reply:
And if the default tag isn't supported, does that mean that Canary upgrades are also not supported for TP1?